test/zippy/mzcompose.py

```diff
@@ -40,9 +40,9 @@
 
 SERVICES = [
     Zookeeper(),
-    Kafka(auto_create_topics=True),
-    SchemaRegistry(),
-    Debezium(),
+    Redpanda(auto_create_topics=True),
+    SchemaRegistry(kafka_servers=[("redpanda", "9092")]),
```

Comment:
We need `redpanda=True` for `SchemaRegistry()` like we have for `Debezium`.

Reply:
Oh wait, no there is no need to run SchemaRegistry at all, it is included in the redpanda container.